### DISCUSSION TITLE
How to handle multiple subscriptions?

### DISCUSSION THREAD

**Discussion by Kralizek** [shibayan/terraform-azurerm-keyvault-acmebot#64] [Q&A]
I have a subscription for production workloads and one for development ones.
I also have a third subscription used for everything that should be shared between these two subscriptions like ACR registries.

I'm planning on installing this module on the third subscription but I wonder if I need to have two distinct instances of the modules to renew the certificates used by the domains created in each subscription.

**Comment by shibayan** [comment]
The current design requires one Acmebot deployment per subscription. If you have separate subscriptions for each environment, deploy to each.
